tests/safeds/data/tabular/containers/_table/test_from_excel_file.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+import openpyxl
+import pytest
+from safeds.data.tabular.containers import Table
+
+from tests.helpers import resolve_resource_path
+
+
+def test_from_excel_file_valid() -> None:
+    path = "./test.xlsx"
+    table = Table.from_dict({"A": ["1"], "B": ["2"]})
+    tmp_table_file = openpyxl.Workbook()
+    tmp_table_file.save(path)
+    try:
+        with Path(path).open("w", encoding="utf-8") as _:
+            table.to_excel_file(path)
+        table = Table.from_excel_file(path)
+        assert table.get_column("A").get_value(0) == 1
+        assert table.get_column("B").get_value(0) == 2
```

Comment:
```suggestion
        table_from_excel = Table.from_excel_file(path)
        assert table_from_excel == table
        assert table_from_excel.get_column("A").get_value(0) == 1
        assert table_from_excel.get_column("B").get_value(0) == 2
```

Reply:
The additional assertions after `assert table_from_excel == table` are not needed, right?

Reply:
This is redundant considering the excel file is now staticly read. (There is no 'table' to compare to)